.circleci/config.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/mern-starter
+    docker:
+      - image: circleci/node:8.11.2
+    steps:
+      - checkout
+      - run:
+          name: update-npm
+          command: 'sudo npm install -g npm@latest'
```

Comment:
sudo?

Reply:
don't know, this is the first time I'm doing a CircleCI 2.0 config. I copied the one they provide for JS projects

Reply:
if you know how to fix, please push the changes